### ISSUE TITLE
original_event.target_agent is 'NoneType'

### ISSUE THREAD

**Issue by LIUQI-creat** [RUC-GSAI/YuLan-Rec#3]
Hi
Thanks for your outstanding work! 
I am reproducing RecAgent and when it executes to **round 4**, I get the following error, how can I fix it? 

```
NFO:9452:David Miller is going to social media.
Traceback (most recent call last):
File "D:\liuqi\LIUQI\yulan-recagent\YuLan-Rec\simulator.py", line 1219, in <module>
main()
File "D:\liuqi\LIUQI\yulan-recagent\YuLan-Rec\simulator.py", line 1210, in main
message = recagent.round()
File "D:\liuqi\LIUQI\yulan-recagent\YuLan-Rec\simulator.py", line 827, in round
msgs = self.one_step(i)
File "D:\liuqi\LIUQI\yulan-recagent\YuLan-Rec\simulator.py", line 607, in one_step
agent2.event = update_event(
File "D:\liuqi\LIUQI\yulan-recagent\YuLan-Rec\utils\event.py", line 39, in update_event
if target_agent in original_event.target_agent:
TypeError: argument of type 'NoneType' is not iterable
```

**Comment by Paitesanshi** [comment]
@LIUQI-creat  Hi, we have addressed this bug in PR #4. You can pull the updated code and verify if the issue has been resolved. Thank you for bringing it to our attention.

**Comment by LIUQI-creat** [comment]
Thank you for the timely resolution! ^^
